nengo_gui/main.py

```diff
@@ -38,7 +38,8 @@ def main():
     parser.add_argument(
         '-b', '--backend', metavar='BACKEND',
         default='nengo', type=str, help='default backend to use')
-    parser.add_argument('--browser', dest='browser', action='store_true')
+    parser.add_argument('--browser', dest='browser', type=str,
+        metavar='BROWSER', default=True, help='browser to use')
```

Comment:
Include some common options in the help string, "(e.g., firefox, google-chrome)"